RELEASE-NOTES.txt

```diff
@@ -1,6 +1,10 @@
 Unreleased
 ---
 
+1.71.0
+---
+* [*] Image block: Replacing the media for an image set as featured prompts to update the featured image [https://github.com/wordpress-mobile/gutenberg-mobile/pull/3930]
```

Comment:
I'm not seeing this image block item in the release notes for the main apps. Was that intentional?

Reply:
I noticed it had already been added to the release notes for the main apps, so there wasn't a need to add it there:

* Android: https://github.com/wordpress-mobile/WordPress-Android/blob/b286a3c327e4e1aada4c9567b972b53afad27611/RELEASE-NOTES.txt#L5
* iOS: https://github.com/wordpress-mobile/WordPress-iOS/blob/a60f3552c4498db902152da331be0c44245c5b96/RELEASE-NOTES.txt#L4